Dockerfile

```diff
@@ -15,7 +15,8 @@ RUN apt-get update && apt-get install -y libmcrypt-dev \
     && docker-php-ext-install zip pdo_mysql mysqli gd sockets \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/* \
-    && rm -rf /tmp/pear
+    && rm -rf /tmp/pear \
+    && mv /var/www/html /var/www/public
```

Comment:
why do we move content from /var/www/html to /var/www/public... up until this point we didn't yet copy any files to the image yet

Reply:
I am only renaming the `html` folder instead of removing it